src/libraries/System.Text.Json/tests/Serialization/NumberHandlingTests.cs

```diff
@@ -1524,12 +1524,12 @@ public static void Attribute_NotAllowed_On_Property_WithCustomConverter()
             string json = @"";
             InvalidOperationException ex = Assert.Throws<InvalidOperationException>(() => JsonSerializer.Deserialize<ClassWith_NumberHandlingOn_Property_WithCustomConverter>(json));
             string exAsStr = ex.ToString();
-            Assert.Contains(typeof(ConverterForInt32).ToString(), exAsStr);
```

Comment:
This info is removed due to simplifying how exceptions are thrown as described in https://github.com/dotnet/runtime/pull/42239/files#r488368843.